[2021-11-27 09:50:51,357] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: daily-scraping.scraping scheduled__2021-11-26T00:00:00+00:00 [queued]>
[2021-11-27 09:50:51,362] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: daily-scraping.scraping scheduled__2021-11-26T00:00:00+00:00 [queued]>
[2021-11-27 09:50:51,362] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-27 09:50:51,362] {taskinstance.py:1242} INFO - Starting attempt 3 of 2
[2021-11-27 09:50:51,362] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-27 09:50:51,369] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): scraping> on 2021-11-26 00:00:00+00:00
[2021-11-27 09:50:51,372] {standard_task_runner.py:52} INFO - Started process 5055 to run task
[2021-11-27 09:50:51,377] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'daily-scraping', 'scraping', 'scheduled__2021-11-26T00:00:00+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/scraping.py', '--cfg-path', '/var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T/tmpieu5aagb', '--error-file', '/var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T/tmpujo14dqz']
[2021-11-27 09:50:51,379] {standard_task_runner.py:77} INFO - Job 232: Subtask scraping
[2021-11-27 09:50:51,409] {logging_mixin.py:109} INFO - Running <TaskInstance: daily-scraping.scraping scheduled__2021-11-26T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-11-27 09:50:51,436] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=daily-scraping
AIRFLOW_CTX_TASK_ID=scraping
AIRFLOW_CTX_EXECUTION_DATE=2021-11-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-26T00:00:00+00:00
[2021-11-27 09:50:51,437] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T
[2021-11-27 09:50:51,438] {subprocess.py:74} INFO - Running command: ['bash', '-c', '/Users/Home/Airflow/tasks/scraping.py /Users/Home/Airflow/down']
[2021-11-27 09:50:51,443] {subprocess.py:85} INFO - Output:
[2021-11-27 09:50:56,413] {local_task_job.py:211} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2021-11-27 09:50:56,418] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 5055
[2021-11-27 09:50:56,419] {taskinstance.py:1411} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-11-27 09:50:56,419] {subprocess.py:99} INFO - Sending SIGTERM signal to process group
[2021-11-27 09:50:56,449] {process_utils.py:66} INFO - Process psutil.Process(pid=5056, status='terminated', started='09:50:51') (5056) terminated with exit code None
[2021-11-27 09:50:56,450] {process_utils.py:66} INFO - Process psutil.Process(pid=5055, status='terminated', exitcode=0, started='09:50:51') (5055) terminated with exit code 0
